### ISSUE TITLE
AdminChooser needs to be replaced with BaseChooser

### ISSUE THREAD

**Issue by Scotchester** [torchbox/wagtailmedia#189]
When using the latest release with Wagtail 4, this warning is thrown:

```
usr/local/lib/python3.8/dist-packages/wagtail/admin/widgets/chooser.py:37: 
RemovedInWagtail50Warning: wagtail.admin.widgets.chooser.AdminChooser is deprecated.
Custom chooser subclasses should inherit from wagtail.admin.widgets.chooser.BaseChooser instead.
```

[`AdminChooser` was replaced by `BaseChooser` in Wagtail 4.0](https://docs.wagtail.org/en/latest/releases/4.0.html#adminchooser-replaced-with-basechooser), so wagtailmedia should be updated here:

https://github.com/torchbox/wagtailmedia/blob/08e06e6fc0c9dca5170aed31d5aebdaac14a4c18/src/wagtailmedia/widgets.py#L26

**Comment by zerolab** [comment]
I believe there's more than that involved in making this happen, but that is the correct place.
PR(s) welcome

**Comment by nickmoreton** [comment]
I noticed that a failing test for [wagtail-webstories](https://github.com/torchbox/wagtail-webstories/pull/20) is because of this import. https://github.com/torchbox/wagtailmedia/blob/cafdf367b0de76c2d6a1c6bf91d2fc57a153f77b/src/wagtailmedia/widgets.py#L10

I'll have a better look around and put in a PR for it.

**Comment by nickmoreton** [comment]
> I'll have a better look around and put in a PR for it.

I see what you mean @zerolab there's more to it. I don't have time for this at the moment 😞

**Comment by zerolab** [comment]
Fixed in #199
